packages/css/src/components/progress-list/progress-list.scss

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@use "../../common/hyphenation" as *;
+@use "../../common/text-rendering" as *;
+
+@mixin reset-ol {
+  list-style-type: none;
+  margin-block: 0;
+  padding-inline-start: 0;
+}
+
+.ams-progress-list__connector {
+  background-color: var(--ams-progress-list-connector-background-color);
+  block-size: 100%;
```

Comment:
Not sure why, but this makes the connector protrude the step container. Replacing this with `align-self: stretch` uses CSS Grid consistently for the layout within the indicator.

It also exposes a bug in connecting the connectors: only the first should have a top margin.